packages/sources/enzyme/test/unit/calcnav.test.ts

```diff
@@ -0,0 +1,40 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { assertError } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../../src'
+import { ENV_RPC_URL } from '../../../curve/src/config'
+
+describe('execute', () => {
+  const jobID = '1'
+  const execute = makeExecute()
+  process.env[ENV_RPC_URL] = process.env[ENV_RPC_URL] || 'http://localhost:8546/'
```

Comment:
Not important but you can also rewrite this as

```js
process.env.ENV_RPC_URL = process.env.ENV_RPC_URL || 'http://localhost:8546/'
```

to keep it consistent with the other tests.